### DISCUSSION TITLE
Different modes for couriers

### DISCUSSION THREAD

**Discussion by wakkytabbaky** [ldtteam/minecolonies#8997] [Ideas]
it would be nice if it was possible to assign a pickup / delivery mode for couriers so that if assigned to delivery they will only fulfill requests and then those set to pickup will run around like normal picking up items from crafting buildings etc.  

or if coding that isnt possible maybe a tool like the foresters zone tool that lets you set which buildings couriers go to / the area they can fulfill orders in.

**Comment by chriskeena** [comment]
It might be useful if the Courier worked a footprint around their home hut and along a path to their assigned warehouse. Opening a new suburb would then be akin to needing to place a post office within that suburb.
